app/src/main/res/layout/activity_detail.xml

```diff
@@ -0,0 +1,119 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Nice that you used `CoordinatorLayout` and `ConstraintLayout` as well! 👏

Reply:
Thank you again! ^^